test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -165,6 +166,20 @@ public void checkDirtyReads() throws Exception {
             assertThatThrownBy(() -> readWriteTxn.putRecord(tableName, key2, null, null))
                     .isExactlyInstanceOf(IllegalArgumentException.class);
         }
+
+        // Validate conflict aborts have full tablename in message
+        try (ManagedTxnContext readWriteTxn = shimStore.tx(someNamespace)) {
+            readWriteTxn.putRecord(table, key1, user_1, user_1);
+            CompletableFuture.runAsync(() -> {
+                ManagedTxnContext anotherTxn = shimStore.tx(someNamespace);
+                anotherTxn.putRecord(table, key1, user_1, user_1);
+                anotherTxn.commit();
+            }).get();
+            readWriteTxn.commit();
+        } catch (TransactionAbortedException ex) {
+            System.out.println(ex);
```

Comment:
nit: Do we need this line `System.out.println`? Breakpoints can be used to check the exception here if required.

Reply:
it was just to share the example of how the line looks like, i will remove it.